frontend/sumo-championship/src/api/club.ts

```diff
@@ -0,0 +1,15 @@
+export const getClubs = (country: string): Promise<string[]> => {
+    //TODO: add here real backend call
```

Comment:
Should we wait for backend with this PR?

Reply:
ye